pages/GEO-Miner.md

```diff
@@ -0,0 +1,10 @@
+The GEO Miner is a placeable machine added in the [GPS](https://github.com/Slimefun/Slimefun4/wiki/GPS) category.
+
+## Obtaining
+The GEO Miner can be crafted in an [Enhanced Crafting Table](https://github.com/Slimefun/Slimefun4/wiki/Enhanced-Crafting-Table).
+
+## Usage
+The GEO Miner requires chunk to be scanned using a [GPS Geo-Scanner](https://github.com/Slimefun/Slimefun4/wiki/GPS-Geo-Scanner) or a [Portable Geo-Scanner](https://github.com/Slimefun/Slimefun4/wiki/Portable-Geo-Scanner).
+</br> The GEO Miner consumes 48 J/s.
+
+The GEO Miner is used to mine scanned resources, mainly [Small Chunk of Uranium](https://github.com/Slimefun/Slimefun4/wiki/Uranium), [Thorium](https://github.com/Slimefun/Slimefun4/wiki/Thorium), [Nether Ice](https://github.com/Slimefun/Slimefun4/wiki/Nether-Ice) and [Salt](https://github.com/Slimefun/Slimefun4/wiki/Salt). This list might be expanded by [addons](https://github.com/Slimefun/Slimefun4/wiki/Addons).
```

Comment:
There is no such thing as "Thorium" in Slimefun, that's probably from a different plugin or addon.